app/javascript/components/greet.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector} from "react-redux";
+import {fetchMessage} from "../redux/reducers/greetSlice";
+
+function Greeting() {
+  const dispatch = useDispatch();
+  console.log('Redux State:', useSelector((state) => state.greeting.message));
+  const message = useSelector(state => state.greeting.message);
+
+  useEffect(() => {
+    dispatch(fetchMessage())
+        .then((data) => {
+          console.log('Fetched Data:', data);
+        })
```

Comment:
- [ ] Kindly remove all `console.log` statements from this file. Leaving `console.log` statements in your files after building applications can be bad because it can potentially expose sensitive information or clutter the console output, impacting code performance and security. 😎 💻